components/automate-ui/src/app/page-components/event-feed-guitar-strings/event-feed-guitar-strings.component.ts

```diff
@@ -390,21 +390,21 @@ export class EventFeedGuitarStringsComponent implements OnInit, OnChanges {
     svg.select('.zoom-slider-button.start')
       .call(d3.drag()
         .on('start', () => { this.dragStarted(); })
-        .on('drag', () => { this.dragStartSlider(); })
+        .on('drag', (e) => { this.dragStartSlider(e); })
         .on('end', () => { this.dragEnded('start'); }));
     svg.select('.zoom-slider-button.end')
       .call(d3.drag()
         .on('start', () => { this.dragStarted(); })
-        .on('drag', () => { this.dragEndSlider(); })
+        .on('drag', (e) => { this.dragEndSlider(e); })
         .on('end', () => { this.dragEnded('end'); }));
   }
 
   dragStarted() {
     this.sliderWidth = this.getComponentWidth();
   }
 
-  dragStartSlider() {
-    const positionPercentage = d3.event.x / this.sliderWidth * 100;
+  dragStartSlider(e) {
+    const positionPercentage = e.x / this.sliderWidth * 100;
```

Comment:
Not familiar with d3, but from looking at patterns in the code (search for `d3.event`) from your prior code, for consistency we should do this:
```suggestion
  dragStartSlider(d3Event: d3.Event) {
    const positionPercentage = d3Event.x / this.sliderWidth * 100;
```

(And similarly in the other functions where you have introduced `e` as a parameter)

Reply:
I noted this below, but to close the loop on this -- When I really started to think about it, I feel it makes more sense to use Native event signatures now, since the with update to Version 6, d3 is actually sending native events now as opposed to previously sending d3 specific events.  I'm glad you brought this up.